sources/api/controllers/CalendarController.ts

```diff
@@ -97,17 +99,34 @@ export async function deleteCalendar(req: Request, res: Response) {
 }
 
 export async function createEvent(req: Request, res: Response) {
+    const requestingUser = getRequestingUser(req);
+    const name = req.body.name;
+    const description = req.body.description;
+    const location = req.body.location;
+    const start_time = req.body.start_time;
+    const end_time = req.body.end_time;
+    const calendar_id = req.params.calendar_id;
+    if (!name || !description || !location || !start_time || !end_time) {
+        throw new ApiException(400, "Bad request");
+    }
+
+    const conn = await DbConnection.getConnection();
+    let calendar = await Calendar.getCalendarWithOwners(conn, calendar_id);
```

Comment:
A new line before and no new line after

Reply:
Also, if we only need to check if a User owns a Calendar, we don't need to fetch the whole owner list.

Just check if the entry (calendar_id, owner_id) exists in `calendars_to_owners`